source/configure/site-configuration-settings.rst

```diff
@@ -329,6 +329,26 @@ iOS app download link
 | String input. Default is ``https://about.mattermost.com/mattermost-ios-app/``.                                                                                                                                        |                                                                        |
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------------------------------------------------------------------+
 
+.. config:setting:: exp-appcustomurlschemes
+  :displayname: App custom URL schemes (Customization)
+  :systemconsole: N/A
+  :configjson: .NativeAppSettings.AppCustomURLSchemes
+  :environment: MM_NATIVEAPPSETTINGS_APPCUSTOMURLSCHEMES
```

Comment:
@cwarnermm - how would you suggest we show how to set this? I don't think it's extremely intuitive to do a space separated list unless instructed